porter/v1/cluster_control_plane.proto

```diff
@@ -219,6 +220,18 @@ message ReadContractResponse {
   string base64_contract = 1;
 }
 
+message ContractComplianceChecksRequest {
+  int64 project_id = 1;
+  int64 cluster_id = 2;
+  // vendor is the name of the vendor to run compliance checks for. Optional, if not provider no vendor checks will be run
+  EnumComplianceVendor vendor = 3;
+}
+
+message ContractComplianceChecksResponse {
+  repeated ContractComplianceCheckGroup check_groups = 1;
+  repeated VendorComplianceCheck vendor_checks = 2;
```

Comment:
I'd appreciate comments explaining these two because I'm honestly not totally sure the difference at first sight